### PR TITLE
Team creation

### PR DESCRIPTION
**Pull request by cameronsuen**
Implemented team creation with unit testing

### CODE REVIEW

File: app/js_controllers/teamCreation.controller.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  };
+
+  $scope.addTeam = function() {
+
+    var newTeamRef = firebase.database().ref().child("team").push();
+    newTeamRef.set({
+      event_uid: $state.event_uid,
+      name: $scope.name,
+      description: $scope.description,
+      skills_needed: $scope.skills,
```
Comment (linhnta925) [inline app/js_controllers/teamCreation.controller.js:40]: Hi Cameron, perhaps you need to load the skills of the "leader" user into the skills_need properties.
Can you do it in future Pull Request?

Other than that, this Pull Request looks great

**Review by linhnta925 — COMMENTED** [review]
Please take a look at my comment below

File: app/js_controllers/teamCreation.controller.js (hunk 1 above)
Comment (cameronsuen) [reply app/js_controllers/teamCreation.controller.js:40]: Ok noted, will do a separate pull request as enhanemment, thanks

**Review by linhnta925 — APPROVED** [review]
Great! I'll merge it

**Pull request merged**